src/linker/Linker/Constants.cs

```diff
@@ -4,8 +4,9 @@
 
 namespace Mono.Linker
 {
-	internal static class Constants
+	static class Constants
 	{
-		public const string ILLinker = "ILLinker";
+		// Prefix used in error/warning reporting to specify the tool it comes from
+		public const string ILLink = "ILLink";
```

Comment:
Please update also documentation references. There are a few places where we use ILLinker